planner/languages/php/php_planner.go

```diff
@@ -52,7 +52,7 @@ func (p *Planner) GetPlan(srcDir string) *plansdk.Plan {
 			fmt.Sprintf("php%s", v.MajorMinorConcatenated()),
 			fmt.Sprintf("php%sPackages.composer", v.MajorMinorConcatenated()),
 		},
-		Definitions: p.definitions(srcDir, v),
+		PackageExtensions: p.pkgExtensions(srcDir, v),
```

Comment:
Should we call these NixLocalVariables instead? extending packages is just one use case. 

You could argue that we should avoid adding stuff here because it makes it harder to do nix free version, but would probably prefer a name that more closely resembles what it represents in the nix file.

Reply:
`NixLocalVariables` would work 👍